.buildkite/e2e-pipeline.yml

```diff
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+agents:
+  provider: "gcp"
+  machineType: "n1-standard-4"
```

Comment:
Let's use `n2-standard-4` because the `n1` family is quite old and gradually having out of capacity issues

Reply:
Addressed in a separate PR: https://github.com/elastic/logstash-filter-elastic_integration/pull/125